### PR TITLE
last-resort: init at 13.001

### PR DESCRIPTION
**Pull request by deviant**
Last Resort is a fallback font that covers every Unicode codepoint.

An example glyph:

image

How this looks in practice:

(Sinhala/Malayalam are covered by system fonts, Telugu/Kannada/Grantham are rendered using Last Resort.)

image

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dschrempf** [comment]
Result of `nixpkgs-review pr 120729` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>last-resort</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10302]: Can you please use deviant?

File: pkgs/top-level/all-packages.nix
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:5962]:
```suggestion
  last-resort = callPackage ../data/fonts/last-resort { };
```

File: maintainers/maintainer-list.nix
Comment (deviant) [reply maintainers/maintainer-list.nix:10302]: No.

Comment (dschrempf) [reply maintainers/maintainer-list.nix:10302]: In my opinion a friendlier reply and a reason for your refusal would have been more appropriate. @SuperSandro2000 does an incredible job as a reviewer and I agree with him, a maintainer with a one letter nick name is harder to find and to track down than maintainers with longer names.

Comment (edef1c) [reply maintainers/maintainer-list.nix:10302]: We have the `github` and `githubId` fields for GitHub usernames. For the sake of all that is holy, don't bother people about their names.
If you want to pick a fight over that, you're welcome to do that with me instead.

Comment (deviant) [reply maintainers/maintainer-list.nix:10302]: @dschrempf

a) You're lacking context. Sandro has previously been petty and rude when reviewing PRs I've made. Please don't jump in and assume things. While I have him blocked, GitHub doesn't actually do anything other than show this to him, filter notification e-mails to me, and hide random pieces of the UI. Since Sandro hasn't taken the hint by now that *maybe* it's impolite to continue harassing someone who has asked them to stop doing so, I think actually giving him a response at all is being more courteous than is necessary.
b) I'm V on IRC. It's really not hard to find me.
c) It's trivial to find my e-mail address from the commit log.
d) If you'd like to convince @v to transfer their handle to me, or convince me to change my IRL name to 'deviant' and handle all the bureaucratic overhead, you're more than welcome to. I'm currently quite happy with my name as-is, though, and don't care about consistency across handles.
e) The list contains GitHub handles and IDs for a reason. The identifiers are simply that: names. V is who I am, not what I am.

I hope you find this explanation satisfactory.

Comment (dschrempf) [reply maintainers/maintainer-list.nix:10302]: @edef1c Thanks for this information. I expressed my opinion in a nice way and by no means wanted to pick a fight. This was solely your interpretation. I was complaining about the briefness and about the lack of friendliness of @deviant towards @SuperSandro2000.

@deviant I didn't know that @SuperSandro2000 has been unfriendly to you about your pull requests, I am sorry. The accusation "harassment" is a very harsh one, though, you may want to think it over. Anyways, how should I have known? I feel like I jumped into a fight the three of you picked some time ago :smile:. From my point of view it was you being unfriendly. Maybe the two/three of you should talk this over in this case? Thanks for your explanation.

I am available at #nixos now, and plan to be more active there in the future, if you want to continue this conversation.

Comment (deviant) [reply maintainers/maintainer-list.nix:10302]: @dschrempf I'm not interested in having this conversation, sorry. Please try to avoid engaging in backseat moderation/tone policing in future. [Willfully ignoring](https://github.com/NixOS/nixpkgs/pull/118712#issuecomment-817898639) being blocked by *multiple people* and choosing to continue to interact with them regardless is harassment *by definition*, not a "harsh" accusation. You didn't *need* to know, as we'd not be conversing presently had you not chosen to jump in earlier. If you think people haven't *repeatedly* tried to talk to him about his behaviour, you're sorely mistaken. There's no fight desired here, just someone lacking in social skills who's being a persistent nuisance and draining whatever motivation I have for interacting with this project. Seriously. Do not push this any further.

Comment (dschrempf) [reply maintainers/maintainer-list.nix:10302]: @deviant I do feel sorry for you and the situation. I do have to express though that I feel attacked by you. I think both, @SuperSandro2000 and you, are doing a great job by supporting the community.

File: pkgs/data/fonts/last-resort/default.nix
Comment (sternenseemann) [inline pkgs/data/fonts/last-resort/default.nix:6]: I wonder if we should maybe call this package `last-resort-font` (or its attribute name?). Upstream uses it for the repo at least and it might make it more discoverable.

**Review by sternenseemann — APPROVED** [review]

File: maintainers/maintainer-list.nix
Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix:10302]: I am not rude as you describe it. I am just short and leave out the pretty fluw and explanations to get more reviews done.

I am also not harassing you or anyone else and I can't remember right now the last conversation with you if we had any at all. I am also not checking everybody's profile if they have blocked me because I don't really care. I am just looking at pull requests and adding notes about mistakes,  improvements and nits to improve the overhaul code quality.

I am also not having any problem when people jump in and continue the conversation by answering questions,  concerns or something else. I am not paid to do this and so it can happen that I take a few days to answer. I am also just a human which tend to forget things unlike computersand the internet.

PS: Next time please just answer that your name was taken on github and you would like to take V. That would have been most likely okay for me and we wouldn't have wasted all our time in a frankly not that productive discussion.

File: pkgs/data/fonts/last-resort/default.nix
Comment (deviant) [reply pkgs/data/fonts/last-resort/default.nix:6]: So, I pondered this: the repo is 'last-resort-font', but the font itself is called 'Last Resort'. nixpkgs doesn't seem to have a consistent pattern (in fact, if anything, it's the opposite: there are a lot of packages that add '-font' to the name attribute, and several where the name ends in '-font' but the attribute doesn't). Additionally, the paths within nixpkgs are sometimes inconsistent (e.g. `tewi-font` is in `pkgs/data/fonts/tewi`).

I'm not opposed to changing this, but do you have any kind of consistent pattern to propose?

Comment (SuperSandro2000) [reply pkgs/data/fonts/last-resort/default.nix:6]: The attr, pname and directory should in most cases match each other except if a suffix/prefix was added for a variant or there is another compelling reason to name it differently. For example the Prometheus exporters are all named in the same pattern despite some of them having a different name in the sources/repositories.

Comment (sternenseemann) [reply pkgs/data/fonts/last-resort/default.nix:6]: > I'm not opposed to changing this, but do you have any kind of consistent pattern to propose?

No, it's fine either way, I'd say.

File: maintainers/maintainer-list.nix
Comment (blaggacao) [reply maintainers/maintainer-list.nix:10302]: > I am not rude as you describe it. I am just short and leave out the pretty fluw and explanations to get more reviews done.

I've had some of your reviews, and they are exactly as you advertise here. I appreciate _speakers_ being short and _listeners_ being kind. Not the other way round. After all, it's our all _time_ and _effectiveness_, that yo hold deer.

Comment (jonringer) [reply maintainers/maintainer-list.nix:10302]: Please act more civil. I don't think @SuperSandro2000 's original suggestion, or other review comments were unwarranted. "V" is not a very unique handle, and I would probably mistake it as a typo. If you're response was, "No, deviant was chosen because V wasn't available", I think the conversation would have ended there.

> Willfully ignoring being blocked by multiple people and choosing to continue to interact with them regardless is harassment by definition, not a "harsh" accusation. You didn't need to know, as we'd not be conversing presently had you not chosen to jump in earlier. If you think people haven't repeatedly tried to talk to him about his behaviour, you're sorely mistaken. There's no fight desired here, just someone lacking in social skills who's being a persistent nuisance and draining whatever motivation I have for interacting with this project. 

I feel like this unwarranted animosity toward Sandro. We should all be aligned with improving nixpkgs, and this and previous comments are not productive conversations.

> There's no fight desired here, just someone lacking in social skills who's being a persistent nuisance and draining whatever motivation I have for interacting with this project. 

At best this is highly dismissive, and at worst, verbal abuse. Please be accepting of other people's differences.

**Comment by IreneKnapp** [comment]
Hi. I'm weighing in as a friend of several people involved, including V, and as a self-appointed representative of the queer community. My own contributions to nixpkgs have been minor, though I'm a long-time user and huge fan. I do not intend to get into any back-and-forth, but I see the potential for this discussion to get out of hand in ways that are, to me, predictable and saddening. I want to do what I can to head that off.

There's a thing that can happen in discussions with people from marginalized backgrounds, where the marginalization means that there are certain things that are abstract discussions to other people, but are a matter of survival to us. Thus, when we defend ourselves, we are often seen as the aggressors, especially by bystanders who don't have context on the issues at hand.

It's already been mentioned that there's a history of conflict between the people involved. I would also like to add that telling us what kinds of names we're allowed to have is a pretty hostile thing, in the context of society today. Speaking as somebody who literally lost everyone in my life at the time when I changed my name, my name is *mine* and I don't acknowledge anybody's right to question it. I don't know the full depth of V's feelings on their name, but I also know that I have no right to ask V to share that pain with me. I'm sharing *my* pain as a proxy, to help people understand, because I have chosen a path in life which involves doing that sort of thing, carefully, when it helps people. Notionally nobody has a right to ask this of me, and it's a shame that I had to step in here, but it's something I freely choose to do because I want to live in a world where this stuff doesn't come up so often.

With that context, then, I very much understand why V reacted that way to being asked to change their name. I understand the argument that brevity can make things efficient, but in this case it made it come off as an order, rather than a request. I personally think, based on the history I've seen, that it actually *was* intended as an order, but I understand that that's highly subjective and impossible to prove.

If we're trying to build an egalitarian community it has to be one based on people *listening*. Both supersandro2000 and V are long-time contributors, and I don't think either of them deserves to be cast solely in a listen-but-don't-object role simply because one has a commit bit and the other doesn't. Listening has to go both ways.

I'm hopeful that my giving context will at least defuse the conversational land-mine, so that the discussion can proceed without any more people stepping on it. If you read this far, thank you.

**Comment by blaggacao** [comment]
No need for a defense, here. :wink: We all can get hurt by one thing or another. :heart: That's human.

**Comment by IreneKnapp** [comment]
Yeah. I think the Discourse side of this discussion is the right place to discuss what comes next, but I wouldn't want it to focus on the name issue. The name issue is context, but the history of conflict is the real problem. I don't have a Discourse account, so I'll say here that in my view the solution needs to be project leadership setting some social norms about how feedback gets handled. That's just my opinion as an outsider, I don't have a strong personal stake in it. I encourage people on the Discourse side of things to continue the conversation from that direction.

File: maintainers/maintainer-list.nix
Comment (nbraud) [reply maintainers/maintainer-list.nix:10302]: > I've had some of your reviews, and they are exactly as you advertise here. I appreciate _speakers_ being short and _listeners_ being kind. Not the other way round. After all, it's our all _time_ and _effectiveness_, that you seem to hold deer.

Then it's fine for V to reply with a simple “no”, right?  They are being short, effective, and mindful of their time.

Comment (blaggacao) [reply maintainers/maintainer-list.nix:10302]: > Then it's fine for V to reply with a simple “no”, right?

No.

---

<sub> ... it leaves you dangling :wink: </sub>

Comment (nbraud) [reply maintainers/maintainer-list.nix:10302]: > > Then it's fine for V to reply with a simple “no”, right?
> 
> No.

I was highlighting the double standard at work here but that seems to be too subtle an approach  >_>'

To be clear: ss2k's replies have the exact same problem; no justification is ever provided, so one is left to either guess or ask what's the actual issue and hope for a more-constructive answer.

Comment (blaggacao) [reply maintainers/maintainer-list.nix:10302]: I also opined about listeners being _kind_. It's important, too.

From the whole context of this thread, I feel the [_good public faith_](https://en.wikipedia.org/wiki/Good_faith) is being under attack and that bothers me (deeply).

I do not understand on what (subjective) basis (any) elements of [malice](https://en.wikipedia.org/wiki/Mens_rea) are interpreted into the context and discussion.

**Comment by jonringer** [comment]
> If we're trying to build an egalitarian community it has to be one based on people _listening_.

I would say a healthy community is founded on mutual understanding.

We don't have to agree on everything, but the PR process is meant to seek criticism for one's work before adding it to the "collective mind share". The verdict doesn't have to be unanimous, but the process should be constructive and inclusive. 

Which is why I think "No" is not an acceptable response, there should be some justification for the question, "Can you please use deviant?". Having the handle of "V" is understandable given a history of using it as your identity, but that's not apparent  between two internet strangers.

I don't think there was an attempt to challenge someone's identity. Just a suggestion that "deviant" would be preferable. It was my preference since it's also the github handle, but I also understand that not everyone gets the account name they wanted.

I would also say there was a lack of trying to understand why sandro was proposing to use the other handle that @deviant had already chosen. It did not need to regress to the state that it did. Text is probably the worse median of communication, as a lot of non-verbal indicators that we take for granted in daily speech are gone. Please just give contributors the benefit of the doubt that they are trying to improve nixpkgs, there's no need to call people a "nuisance" or refer to reasonable PR comments as "harassment".

File: maintainers/maintainer-list.nix
Comment (Profpatsch) [reply maintainers/maintainer-list.nix:10302]: Names are not up for discussion.

Comment (Profpatsch) [reply maintainers/maintainer-list.nix:10302]: I am kind of dismayed by people seriously telling other people that they “can’t have a name”.

**Comment by alyssais** [comment]
> Please just give contributors the benefit of the doubt that they are trying to improve nixpkgs, there's no need to call people a "nuisance" or refer to reasonable PR comments as "harassment".

I am appalled at how you are deliberately missing the point here and refusing to engage at all with the concerns that have been raised. (I say deliberately because I _know_ other people have spoken to you about their concerns about Sandro's behaviour.)

Sandro has been persistently rude and even **block evaded**, and V is by far the only person to have raised issues with it.

File: maintainers/maintainer-list.nix
Comment (alyssais) [reply maintainers/maintainer-list.nix:10302]: > From the whole context of this thread, I feel the good faith is being under attack and that bothers me (deeply).
>
> I do not understand on what (subjective) basis (any) elements of malice are interpreted into the context and discussion.

Perhaps this is an indication that you are missing that context, and that you shouldn't be trying to judge interactions between two people who clearly have a history from the peanut gallery.

Comment (dschrempf) [reply maintainers/maintainer-list.nix:10302]: @Profpatsch I did not try to tell V to not have a specific name. I just expressed my opinion about the nature of the conversation and about the name. However, I was very dismayed, to use your word, at the course of the discussion that followed. I should have expressed my opinion in a more appropriate way, but then, I can also expect to get a more proper, and less emotional, response.

Comment (primeos) [reply maintainers/maintainer-list.nix:10302]: > I am kind of dismayed by people seriously telling other people that they “can’t have a name”.

Agreed but not sure if anyone actually said that. "Can you please use deviant?" is just a suggestion/question and due to https://github.com/deviant/nixpkgs/blob/0f01e4f1b959f49897f94c485e71069bd923cc19/maintainers/maintainer-list.nix#L27 that question makes IMO sense. But I can (more or less) understand all parties here and this is not me saying that `V` is bad or "wrong" but that, looking at this discussion, we might want to drop https://github.com/deviant/nixpkgs/blob/0f01e4f1b959f49897f94c485e71069bd923cc19/maintainers/maintainer-list.nix#L27 (not sure if this is also stated in other places) to avoid issues like this in the future.

**Comment by blaggacao** [comment]
> Sandro has been persistently rude and even block evaded, and V far from the only person to have raised issues with it.

Wait a second, nobody has cited sufficient proof for any of those accusations.

Without further proof, it almost appears to me that cancel culture is trying to be played out, here. No offense, if I'm wrong and please help me understand your cause.

But the _fundamental and blanco waiver_ of the assumption of good faith is really concerning for a healthy community communication.

I would be comforted, if we all could agree to pledge on the foundations of good faith.

**Comment by alyssais** [comment]
Okay, here's proof of deliberate block evasion:

- https://github.com/NixOS/nixpkgs/pull/118712#issuecomment-817898639
- https://logs.nix.samueldr.com/nixos-dev/2021-04-04#4817600;

**Comment by Mic92** [comment]
> Okay, here's proof of deliberate block evasion:
> 
>     * [#118712 (comment)](https://github.com/NixOS/nixpkgs/pull/118712#issuecomment-817898639)
> 
>     * https://logs.nix.samueldr.com/nixos-dev/2021-04-04#4817600;

Sorry, but he just wanted to explain the semantics of the rebase action here... I fail to see the problem here.

**Comment by nrdxp** [comment]
Seems to me he is being a little sarcastic in that IRC, and is frustrated with having to make a political intervention on a technical project, which I can totally understand tbh. To me, it's gonna take a little more than that to prove bad faith in someone who has been utterly prolific in reviewing and merging PRs.

**Comment by jonringer** [comment]
I don't claim to know the full story, I'm only aware of [one other argument](https://github.com/NixOS/nixpkgs/pull/110060#discussion_r560733506). People ignoring each other obviously signals a failure of communication, and it's not a healthy practice; I wished things occurred differently.

> I say deliberately because I _know_ other people have spoken to you about their concerns about Sandro's behaviour.

Yes, it has been voiced to me. And I've talked to him, and I would say that his review behavior has improved since January. Is there still room for improvement? always. Are lot of issues from January resolved or mitigated? I would say so.

Largely I only have insight to PR's that we both review, and a lot of his comments come off as "nitpicky", but I think they are valid in the majority of cases. He comments on my PRs, and I would rather have someone give me a few too many suggestions, than get no reviews at all.

In context of this PR, I don't think there was any malicious intent in sandro's comments. People have multiple handles, and it was just suggested to use the github handle.

I know a lot of people are emotionally invested into this issue, so it's hard to be objective. But I don't see any initial intention to spark further controversy, at least in context of this PR; however, that's what came to pass.

> Okay, here's proof of deliberate block evasion:
> 
>     * [#118712 (comment)](https://github.com/NixOS/nixpkgs/pull/118712#issuecomment-817898639)

Seems like he's trying to make progress on the PR and get it merged. Was the comment about being blocked necessary? no. But rebasing the branch to correct base branch seems like the right action.

>     * https://logs.nix.samueldr.com/nixos-dev/2021-04-04#4817600;

The surrounding conversation is about python changes, seems odd and out of place.

**Comment by alyssais** [comment]
I was asked to provide proof of block evasion.  I did so.  It appears that now the goalposts have shifted from proving block evasion, to arguing whether even though there was block evasion, maybe this was somehow acceptable block evasion.

The problem is not with anything that he said, it's that he continues to interact with people he knows have blocked him.  There is a repeating pattern where he annoys or upsets somebody to the point where they make it clear they don't want to interact with him any more, and he doesn't back off.  The other people involved have been doing the responsible thing in trying to limit their interactions with him to avoid escalating, but by refusing to respect that, Sandro is making de-escalation impossible.

The comments about rebasing, or about Python, are not wrong.  But they have to be seen in the context they are in -- continuing to try to engage with somebody who has expressed a clear preference otherwise.  And none of these comments are things that only Sandro was capable of making.  If he hadn't made them, it would probably only have been a few hours until somebody else did.  There was _no need_ to deliberately ignore other people's boundaries.

I don't think that anybody is acting in bad faith here.  But I think Sandro is engaging in harmful behaviour, and he has shown no indication of being willing to understand _why_ his behaviour is harmful, or to be willing to improve.  Any time he has been offered feedback (that I've seen), the response has either been ["I don't care"](https://github.com/NixOS/nixpkgs/pull/120729#discussion_r622130384), ["I'm lazy"](https://logs.nix.samueldr.com/nixos/2021-04-14#4858713;), or outright dismissal.

Why do you think so many people have him blocked that this is a recurring problem?

**Comment by nrdxp** [comment]
If you don't believe he is acting in bad faith then what's the issue? People are who they are. I never once saw sandro's comments as rude. I tended to view them as concise and to the point. If you want to construe that as rude that's totally up to you and that's fine, but if he chooses to blatantly ignore you for doing so, that's also fine.

Consider it from his position. What if I started trying to tell you that your being too sensitive and need to let it go (which I'm not doing by the way)? What if I then went as far as to rally a bunch of people who agree with me and start repetively addressing it to you. You might think that my opinion has nothing to do with the work we are trying to accomplish here, and blatantly ignore it. And you know what, you'd be right to do so in my opinion.

**Comment by 7c6f434c** [comment]
> Sorry, but he just wanted to explain the semantics of the rebase action here... I fail to see the problem here.

I agree about good faith (whether or not it is a good idea). But this is definitely enough to substantiate that mentioning block evasion is also done in good faith.

File: maintainers/maintainer-list.nix
Comment (jonringer) [reply maintainers/maintainer-list.nix:10302]: > I am kind of dismayed by people seriously telling other people that they “can’t have a name”.

I don't think that was the case at all, sandro suggested the github handle. Everyone agrees there's no technical reason to disallow a name like "V", just a subjective preference. Personally, I find it "easier" to mention someone to do `@<maintainer>` than `@$(nix eval .#lib.maintainers.<maintainer>.github)`, which is why personally I would prefer the github handle (Also, with a single letter, I'll probably have some additional cognitive dissonance about whether it's correct and have to verify). However, there's always the possibility of nixpkgs moving away from github, and trying to keep alignment with github would be irrelevant anyway.

Anyway, from a largely third party perspective, there's is discontinuity between the suggestion to use the github handle and everything that followed.

**Comment by alyssais** [comment]
I don't think we're getting anywhere here, and like Jon says,

> Text is probably the worse median of communication, as a lot of non-verbal indicators that we take for granted in daily speech are gone.

So I'd like to try something else, in the hopes it can help us to emathize with each other's positions and come up with a shared understanding to use to move forward. (Or at least to discover that it's not possible to come up with such a thing in a way that is less public, and hopefully less hostile.)

For the next couple of hours at least, I'll be in this Jitsi room:

https://jitsi.nixcon.org/AudioArtsCorrectGloriously

**Comment by blaggacao** [comment]
> I was asked to provide proof of block evasion.

I stated that you made _accusations_ and did not provide proof (haramment, rude).

_block evasion_ is no accusation, it's not even inherently bad, if not done with malice, but for practical reasons (because of an objective need for communication).

This here is a public space, and figuratively speaking, _nobody_ can rightfulky ask for _anybody_ to leave the scene before they enter.

Block evasion is meant so that you can exclude people from entering your "home zone", those are two completely different things.

**Comment by deviant** [comment]
Hello. I just opened my mail client today and this is what I am faced with.

why

I specifically unsubscribed from this issue so I wouldn't have to get it pushed in my face again, but apparently someone tagged me again.

This is why I find it exhausting to try to contribute to nixpkgs. To whomever wants to tell me otherwise, please take a long, hard look at yourself

Should people be expected to deal with this shit when they're trying to add a package, contribute a fix, etc? Really?

And for the love of god, open a new issue

**Comment by deviant** [comment]
@grahamc can you please lock this

**Comment by Pacman99** [comment]
These discussions are good, but this PR thread is a really bad place to have them. The conversation at the start is purely between @deviant and @SuperSandro2000 one has the right to block another and say their reasons for doing so, the other has the right to keep reviewing PRs and not be required to keep track of everyone that has blocked them

Its not great that we took a conversation that was between two people and made it about ourselves. If you want to share your experience thats amazing, but I think it should be done on a thread where the intention is to have these discussions. Perhaps the community manifesto thread would be a better place.

**Comment by deviant** [comment]
I genuinely want nothing more than to be left alone. The entire conversation that led up to someone publicly posting about this consisted of me asking to be left alone. How difficult is it to understand that when someone asks you not to do a thing, the correct thing is not to fucking do *the opposite but more*

**Comment by alyssais** [comment]
I've locked the thread because of the request not to involve V in further conversations.  Now, because of the way locking works, many of us can still post in the thread, because we're collaborators on the repository.  So all I can do is ask everybody else here to respect V's desire to be left out of this conversation, and to move it somewhere else.

**Pull request merged**
